docs/ddp/README.md

```diff
@@ -0,0 +1,186 @@
+
+
+# SR-IOV network device plugin with DDP
+Dynamic Device Personalization aka DDP allows dynamic reconfiguration of the packet processing pipeline of Intel Ethernet 700 Series to meet specific use case needs on demand, adding new packet processing pipeline configuration *profiles* to a network adapter at run time, without resetting or rebooting the server.
```

Comment:
I think it is possible, since the profile is added to the PF. And for x700 you do not need to reload the driver to load a new profile. So, existing VFs are unaffected.

Reply:
In this case, will existing VFs be able to use the dynamically loaded new profile?
btw, if VFs from one PF are using different dirvers (e.g. native kernel driver or vfio-pci driver), then the profile can be used by both types of VF devices, right?

Reply:
The DDP profile is firmware on the adapter itself. So, any profile that is loaded into the card that is applicable for all VF exist on that card, regardless of VF drivers.

Reply:
Okay, VF device will not be aware of the change in firmware, so if ddp profile is re-loaded, then SR-IOV device plugin will also need to be restarted in order to detect new profile names.

Reply:
That's right.